ow_lander/src/ow_lander/actions.py

```diff
@@ -86,8 +88,11 @@ class ArmStopServer(mixins.ArmActionMixin, ActionServerBase):
   result_type   = owl_msgs.msg.ArmStopResult
 
   def execute_action(self, _goal):
+    # Reset faults messages before the arm start moving
+    self._arm_faults.reset_arm_faults_flags()
     if self._arm.stop_arm():
       self._set_succeeded("Arm trajectory stopped")
+      self._arm_faults.set_arm_faults_flag(ArmFaultsStatus.E_STOP)
```

Comment:
I don't think we have actually agreed we would implement E_STOP as a fault flag, and I don't know if it's right to call our stop command an "emergency stop". @kmdalal does this make sense? Should we keep it?

Reply:
My thoughts:
 - I assume this refers to the ArmStop action.
 - That action should cause a goal error in the action that was stopped.
 - E_STOP is an OWLAT hardware concept (a physical stop button) and I don't think we'll ever model it in OceanWATERS.

If some discussion would be helpful, I'm available Friday morning.

Reply:
A goal error should already occur when the arm is stopped, although whether it is a `TASK_GOAL_ERROR` or `ARM_GOAL_ERROR` depends on the action that was stopped. 

So it sounds to me like we do not want an arm stop to tick the `E_STOP` flag in `/arm_fault_status`?